crates/core/tedge/src/cli/certificate/upload.rs

```diff
@@ -71,30 +72,21 @@ impl UploadCertCmd {
             self.password.clone()
         };
 
-        let config = TEdgeConfig::try_new(TEdgeConfigLocation::default())?;
-        let root_cert = &config.c8y.root_cert_path;
-        let client_builder = reqwest::blocking::Client::builder();
-        let res = match std::fs::metadata(root_cert) {
-            Ok(res) => res,
-            Err(e) => match e.kind() {
-                std::io::ErrorKind::NotFound => {
-                    return Err(CertError::RootCertificatePathDoesNotExist(
-                        root_cert.to_string(),
-                    ))
+        let mut client_builder = reqwest::blocking::Client::builder();
```

Comment:
Wondering why we're not using the `RootCertClient` here.

Reply:
It's because we're using the blocking client here, I forgot about that because I made those changes a while ago